README.md

```diff
@@ -128,12 +128,9 @@ kubectl delete --ignore-not-found customresourcedefinitions \
 
 ### Testing
 
-> Ensure that you're running tests in the following path:
-> `$GOPATH/src/github.com/coreos/prometheus-operator` as tests expect paths to
-> match. If you're working from a fork, just add the forked repo as a remote and
-> pull against your local coreos checkout before running tests.
+#### Unit tests
 
-#### Running *unit tests*:
+For short unit tests run:
```

Comment:
Maybe instead we can do:
`#### Quickstart`

Reply:
Not sure if I follow. We already have `## Quickstart` section. (: what we are trying to improve here?

Reply:
For short unit tests, does not make that much sense to me. The wording. I meant more like quick start with unit tests? But its a nit, I am fine with leaving it like this.